src/components/study-pane.js

```diff
@@ -565,6 +609,35 @@ const StudyPane = (props) => {
         // Note: studyUuid and dispatch don't change
     }, [studyUuid, dispatch]);
 
+    const loadTree = useCallback(() => {
+        console.info(
+            `Loading network modification tree of study '${studyUuid}'...`
+        );
+
+        const networkModificationTree = fetchNetworkModificationTree(studyUuid);
+
+        networkModificationTree
+            .then((tree) => {
+                const networkModificationTreeModel =
+                    new NetworkModificationTreeModel();
+                networkModificationTreeModel.setTreeElements(tree);
+                networkModificationTreeModel.updateLayout();
+                dispatch(
+                    loadNetworkModificationTreeSuccess(
+                        networkModificationTreeModel
+                    )
+                );
+            })
+            .catch(function (error) {
+                console.error(error.message);
+                setStudyNotFound(true);
```

Comment:
show the real error

Reply:
Accurate error message is now displayed in snackbar